org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/window.css.dsp

```diff
@@ -153,3 +153,9 @@ input[type="checkbox"]:focus
 	font-weight: bold;
 	margin: 5px;
 }
+
+.date-picker-calendar-button {
+	position: absolute;
+    right: 0px;
+    top: 5px;
+}
```

Comment:
again ... please update idempiere.theme.default in lang-addon.xml when you  make changes to css ....